Packs/Core/Playbooks/playbook-Uncommon_remote_scheduled_task_created.yml

```diff
@@ -0,0 +1,1352 @@
+    taskid: 7272972f-d88b-484d-897b-61c0fce7def0
+    type: regular
+    task:
+      id: 7272972f-d88b-484d-897b-61c0fce7def0
+      version: -1
+      name: Determine whether the remote IP address is internal or external
+      description: Returns yes if the IP is in one of the ranges provided, returns no otherwise.
+      scriptName: IsIPInRanges
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "2"
+    scriptarguments:
+      ip:
+        complex:
+          root: Core.OriginalAlert.event.actor_remote_ip
+          filters:
+          - - operator: notContainsGeneral
+              left:
+                value:
+                  simple: Core.OriginalAlert.event.actor_remote_ip
+                iscontext: true
+              right:
+                value:
+                  simple: '::'
+              ignorecase: true
+      ipRanges:
+        complex:
+          root: lists
+          accessor: PrivateIPs
+          transformers:
+          - operator: RegexExtractAll
+            args:
+              error_if_no_match: {}
+              ignore_case: {}
+              multi_line: {}
+              period_matches_newline: {}
+              regex:
+                value:
+                  simple: (\b(?:\d{1,3}\.){3}\d{1,3}\b/\d{1,2})
+              unpack_matches: {}
+          - operator: join
+            args:
+              separator:
+                value:
+                  simple: ','
+    separatecontext: false
+    continueonerror: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 410,
+          "y": 180
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "41":
+    id: "41"
+    taskid: 5ba5e082-b8f3-413f-89f6-40261ef6a811
+    type: title
+    task:
+      id: 5ba5e082-b8f3-413f-89f6-40261ef6a811
+      version: -1
+      name: Analyst Decision
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "42"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1070,
+          "y": 2030
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "42":
+    id: "42"
+    taskid: 951c96f5-d9b9-4a75-802a-da1d01d66810
+    type: condition
+    task:
+      id: 951c96f5-d9b9-4a75-802a-da1d01d66810
+      version: -1
+      name: Analyst decision for suspicious cmd parameters
+      description: Analyst review is required to determine, based on suspicious command-line parameters, whether to take remediation actions  such as disabling the scheduled task and blocking the IP if malicious or to close the alert as a false positive.
```

Comment:
Based on suspicious command line parameters, analyst review is required to determine whether to take remediation actions such as disabling the scheduled task and blocking the IP if malicious, or to close the alert as a false positive.